components/navigable-container/README.md

```diff
@@ -0,0 +1,156 @@
+NavigableContainer
+=============
+
+`NavigableContainer` is a React component to render a container navigable using the keyboard. Only things that are focusable can be navigated to. It will currently always be a `div`.
+
+`NavigableContainer` is exported as three convenience classes: `NavigableMenu`, `NavigableGrid` and `TabbableContainer`. `NavigableContainer` itself is not exported. `NavigableMenu`, `NavigableGrid`, and `TabbableContainer` have some shared props, and some individual props. Any other props will be passed through to the `div`.
+
+----
+
+## Shared Props
+
+These are the props that `NavigableMenu`, `NavigableGrid` and `TabbableContainer` all share.
+
+
+### initialSelector
+
+A selector which tells the component what to focus first inside it. If the selector is not matched, the first focusable item will be focused. This can be useful to focus the previously selected descendant when this component get focused.
```

Comment:
I tend to avoid "selector" props because they break quickly without noticing. 
Is there a way we could remove this prop entirely, say by always focusing the first tabbable or something?

--

Alternatively, we could try to always fallback to the tabbable with `aria-selected` if present

Reply:
I've liked being able to use classes for things which have 'fake focus' in the past, but it's easy enough to add later I suppose if we need it. The `aria-selected` selector is definitely currently needed in TabPanel (you don't want to just focus the first tab if one was previously selected), so I guess we could look for just that first and fallback to the first one as you say. I tend to over-engineer rather than under-engineer, so you're just seeing that excess :)